tracee-ebpf/tracee/co-re/missing_definitions.h

```diff
@@ -49,6 +49,19 @@
 #define sk_flags          __sk_common.skc_flags
 #define sk_rxhash         __sk_common.skc_rxhash
 
+#define IPPROTO_HOPOPTS		0	/* IPv6 hop-by-hop options	*/
```

Comment:
Could you squash changes to missing_definitions into a single commit and pay attention to numbers being aligned and separated from the macro by spaces ?

```C
#define IPPROTO_ICMPV6          58

#define ICMP_ECHO               8
#define ICMP_EXT_ECHO           42

#define ICMPV6_ECHO_REQUEST     128
```